{{cookiecutter.repository_name}}/operations/utilities/contested_display.py

```diff
@@ -0,0 +1,43 @@
+import operations
```

Comment:
I guess this split is because we need to hack around guild

I checked now that gradio does not need a hack like this if we want to run the program through guild. It is extremely similar to streamlit but more popular